packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -483,6 +498,27 @@ export default class Editor implements IEditor {
         return !!range && this.contains(range) && core.api.selectRange(core, range);
     }
 
+    private getImageSelection(
+        core: EditorCore,
+        arg: Range | NodePosition | Node | SelectionPath | HTMLTableElement | null
+    ) {
+        if (safeInstanceOf(arg, 'HTMLImageElement')) {

```

Comment:
Also need to check if arg2 is undefined.

Because if I call select(image, PositionType.Before), it is still not an image selection

Reply:
I think one problem here is how we can distinguish if we want an image selection, or a normal selection with just a IMG tag? Both of them have some scenario to use.

Maybe we need some additional parameter. Or, let this API support ImageSelectionRange (as well as TableSelectionRange) so that we can check the object type

Reply:
If we select something else and image it will be recognize as normalSelection, because the arg will be the parent node the image and something else.